lib/fog/sakuracloud/models/volume/disk.rb

```diff
@@ -38,6 +38,11 @@ def carve_hostname_on_disk(hostname)
           true
         end
 
+        def attach(server_id)
+          service.attach_disk(@attributes[:id], server_id )
```

Comment:
Thank you for reviewing.

I followed the same style above.
It uses @attributes a lot.
Do I use `id` just for this method?

```
        def save
          requires :name, :plan, :source_archive
          data = service.create_disk(@attributes[:name], @attributes[:plan], @attributes[:source_archive]).body["Disk"]
          merge_attributes(data)
          true
        end

        def configure(sshkey_id)
          requires :id
          service.configure_disk(@attributes[:id], sshkey_id )
          true
        end

        def carve_hostname_on_disk(hostname)
          requires :id
          service.carve_hostname_on_disk(@attributes[:id], hostname )
          true
        end

        def attach(server_id)
          service.attach_disk(@attributes[:id], server_id )
          true
        end
```